File: Selenium/activity1.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class activity1 {

	public static void main(String[] args) throws InterruptedException {

		WebDriver driver = new ChromeDriver();

		driver.get("https://training-support.net/");

		System.out.println("Title of the Page is " + driver.getTitle());

		driver.findElement(By.linkText("About Us")).click();

		System.out.println("About Us Page Title: " + driver.getTitle());

		driver.quit();

	}

}

File: Selenium/activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC


driver = webdriver.Chrome()
wait = WebDriverWait(driver, 10)


driver.get("https://training-support.net")


print("Home Page Title:", driver.title)


about_us = wait.until(
    EC.element_to_be_clickable((By.LINK_TEXT, "About Us"))
)
about_us.click()

print("About Us Page Title:", driver.title)


driver.quit()
